28.03.2021 12:46:21 : Дешифрование пароля успешно завершено
28.03.2021 12:46:36 : Хеш-последовательность успешно получена
28.03.2021 12:46:36 : Хеш-последовательность успешно получена
28.03.2021 12:46:36 : Дешифрование пароля успешно завершено
28.03.2021 12:47:04 : System.Exception: Не найдена папка
   в SZMK.Desktop.Views.KB.KB_DetailedInformationOrder_F.PathArhive_TB_MouseDoubleClick(Object sender, MouseEventArgs e) в C:\Users\thedr\Desktop\Works\SZMK\SZMK-Software-Package\SZMK.Desktop\Views\KB\KB_DetailedInformationOrder_F.cs:строка 147
28.03.2021 12:48:56 : System.Exception: Не найдена папка
   в SZMK.Desktop.Views.KB.KB_DetailedInformationOrder_F.PathDetailsDWG_TB_MouseDoubleClick(Object sender, MouseEventArgs e) в C:\Users\thedr\Desktop\Works\SZMK\SZMK-Software-Package\SZMK.Desktop\Views\KB\KB_DetailedInformationOrder_F.cs:строка 67
28.03.2021 12:49:43 : Дешифрование пароля успешно завершено
28.03.2021 12:49:50 : Хеш-последовательность успешно получена
28.03.2021 12:49:50 : Хеш-последовательность успешно получена
28.03.2021 12:49:50 : Дешифрование пароля успешно завершено
28.03.2021 12:50:45 : Дешифрование пароля успешно завершено
28.03.2021 12:50:49 : Хеш-последовательность успешно получена
28.03.2021 12:50:49 : Хеш-последовательность успешно получена
28.03.2021 12:50:50 : Дешифрование пароля успешно завершено
28.03.2021 12:51:17 : System.Management.ManagementException: Не найдено 
   в System.Management.ManagementException.ThrowWithExtendedInfo(ManagementStatus errorCode)
   в System.Management.ManagementObject.Initialize(Boolean getObject)
   в System.Management.ManagementBaseObject.get_Properties()
   в System.Management.ManagementBaseObject.GetPropertyValue(String propertyName)
   в System.Management.ManagementBaseObject.get_Item(String propertyName)
   в SZMK.Desktop.Views.KB.KB_DetailedInformationOrder_F.ChangePathDetailsDWG_B_Click(Object sender, EventArgs e) в C:\Users\thedr\Desktop\Works\SZMK\SZMK-Software-Package\SZMK.Desktop\Views\KB\KB_DetailedInformationOrder_F.cs:строка 208
28.03.2021 12:54:21 : Дешифрование пароля успешно завершено
28.03.2021 12:54:31 : Хеш-последовательность успешно получена
28.03.2021 12:54:31 : Хеш-последовательность успешно получена
28.03.2021 12:54:32 : Дешифрование пароля успешно завершено
28.03.2021 12:54:43 : System.Exception: Не найдена папка
   в SZMK.Desktop.Views.KB.KB_DetailedInformationOrder_F.ChangePathDetailsDWG_B_Click(Object sender, EventArgs e) в C:\Users\thedr\Desktop\Works\SZMK\SZMK-Software-Package\SZMK.Desktop\Views\KB\KB_DetailedInformationOrder_F.cs:строка 199
28.03.2021 12:54:50 : System.Management.ManagementException: Не найдено 
   в System.Management.ManagementException.ThrowWithExtendedInfo(ManagementStatus errorCode)
   в System.Management.ManagementObject.Initialize(Boolean getObject)
   в System.Management.ManagementBaseObject.get_Properties()
   в System.Management.ManagementBaseObject.GetPropertyValue(String propertyName)
   в System.Management.ManagementBaseObject.get_Item(String propertyName)
   в SZMK.Desktop.Views.KB.KB_DetailedInformationOrder_F.ChangePathDetailsDWG_B_Click(Object sender, EventArgs e) в C:\Users\thedr\Desktop\Works\SZMK\SZMK-Software-Package\SZMK.Desktop\Views\KB\KB_DetailedInformationOrder_F.cs:строка 211
28.03.2021 12:54:54 : System.Management.ManagementException: Не найдено 
   в System.Management.ManagementException.ThrowWithExtendedInfo(ManagementStatus errorCode)
   в System.Management.ManagementObject.Initialize(Boolean getObject)
   в System.Management.ManagementBaseObject.get_Properties()
   в System.Management.ManagementBaseObject.GetPropertyValue(String propertyName)
   в System.Management.ManagementBaseObject.get_Item(String propertyName)
   в SZMK.Desktop.Views.KB.KB_DetailedInformationOrder_F.ChangePathDetailsDWG_B_Click(Object sender, EventArgs e) в C:\Users\thedr\Desktop\Works\SZMK\SZMK-Software-Package\SZMK.Desktop\Views\KB\KB_DetailedInformationOrder_F.cs:строка 211
28.03.2021 12:54:58 : System.Management.ManagementException: Не найдено 
   в System.Management.ManagementException.ThrowWithExtendedInfo(ManagementStatus errorCode)
   в System.Management.ManagementObject.Initialize(Boolean getObject)
   в System.Management.ManagementBaseObject.get_Properties()
   в System.Management.ManagementBaseObject.GetPropertyValue(String propertyName)
   в System.Management.ManagementBaseObject.get_Item(String propertyName)
   в SZMK.Desktop.Views.KB.KB_DetailedInformationOrder_F.ChangePathDetailsDWG_B_Click(Object sender, EventArgs e) в C:\Users\thedr\Desktop\Works\SZMK\SZMK-Software-Package\SZMK.Desktop\Views\KB\KB_DetailedInformationOrder_F.cs:строка 211
28.03.2021 12:56:02 : Дешифрование пароля успешно завершено
28.03.2021 13:17:30 : Дешифрование пароля успешно завершено
28.03.2021 13:17:46 : Дешифрование пароля успешно завершено
28.03.2021 13:17:51 : Хеш-последовательность успешно получена
28.03.2021 13:17:51 : Хеш-последовательность успешно получена
28.03.2021 13:17:51 : Дешифрование пароля успешно завершено
28.03.2021 13:19:22 : Дешифрование пароля успешно завершено
28.03.2021 13:19:30 : Хеш-последовательность успешно получена
28.03.2021 13:19:30 : Хеш-последовательность успешно получена
28.03.2021 13:19:30 : Дешифрование пароля успешно завершено
28.03.2021 13:20:12 : Дешифрование пароля успешно завершено
28.03.2021 13:20:19 : Хеш-последовательность успешно получена
28.03.2021 13:20:19 : Хеш-последовательность успешно получена
28.03.2021 13:20:19 : Дешифрование пароля успешно завершено
28.03.2021 13:21:56 : Дешифрование пароля успешно завершено
28.03.2021 13:22:01 : Хеш-последовательность успешно получена
28.03.2021 13:22:01 : Хеш-последовательность успешно получена
28.03.2021 13:22:01 : Дешифрование пароля успешно завершено
28.03.2021 13:55:47 : Дешифрование пароля успешно завершено
28.03.2021 13:55:52 : Хеш-последовательность успешно получена
28.03.2021 13:55:52 : Хеш-последовательность успешно получена
28.03.2021 13:55:52 : Дешифрование пароля успешно завершено
28.03.2021 13:55:57 : System.Exception: System.NullReferenceException: Ссылка на объект не указывает на экземпляр объекта.
   в SZMK.Desktop.Services.Request.GetAllOrders() в C:\Users\thedr\Desktop\Works\SZMK\SZMK-Software-Package\SZMK.Desktop\Services\Request.cs:строка 1061 ---> System.Exception: System.NullReferenceException: Ссылка на объект не указывает на экземпляр объекта.
   в SZMK.Desktop.Services.Request.GetAllOrders() в C:\Users\thedr\Desktop\Works\SZMK\SZMK-Software-Package\SZMK.Desktop\Services\Request.cs:строка 1061 ---> System.Exception: System.NullReferenceException: Ссылка на объект не указывает на экземпляр объекта.
   в SZMK.Desktop.Services.Request.GetAllOrders() в C:\Users\thedr\Desktop\Works\SZMK\SZMK-Software-Package\SZMK.Desktop\Services\Request.cs:строка 1061
   в SZMK.Desktop.Services.Request.GetAllOrders() в C:\Users\thedr\Desktop\Works\SZMK\SZMK-Software-Package\SZMK.Desktop\Services\Request.cs:строка 1117
   в SZMK.Desktop.Services.RequestLinq.GetData() в C:\Users\thedr\Desktop\Works\SZMK\SZMK-Software-Package\SZMK.Desktop\Services\RequestLinq.cs:строка 307
   --- Конец трассировки внутреннего стека исключений ---
   в SZMK.Desktop.Services.RequestLinq.GetData() в C:\Users\thedr\Desktop\Works\SZMK\SZMK-Software-Package\SZMK.Desktop\Services\RequestLinq.cs:строка 313
   в SZMK.Desktop.Services.RequestLinq.GetOrdersUser(INotifyProcess notifier) в C:\Users\thedr\Desktop\Works\SZMK\SZMK-Software-Package\SZMK.Desktop\Services\RequestLinq.cs:строка 75
   --- Конец трассировки внутреннего стека исключений ---
   в SZMK.Desktop.Services.RequestLinq.GetOrdersUser(INotifyProcess notifier) в C:\Users\thedr\Desktop\Works\SZMK\SZMK-Software-Package\SZMK.Desktop\Services\RequestLinq.cs:строка 114
   в SZMK.Desktop.Views.KB.KB_F.GetOrders(Int32 ForViews, ForLongOperations_F Dialog) в C:\Users\thedr\Desktop\Works\SZMK\SZMK-Software-Package\SZMK.Desktop\Views\KB\KB_F.cs:строка 890
   в SZMK.Desktop.Views.KB.KB_F.RefreshOrder(Int32 ForViews, ForLongOperations_F Dialog) в C:\Users\thedr\Desktop\Works\SZMK\SZMK-Software-Package\SZMK.Desktop\Views\KB\KB_F.cs:строка 872
28.03.2021 13:57:52 : Дешифрование пароля успешно завершено
28.03.2021 13:57:58 : Хеш-последовательность успешно получена
28.03.2021 13:57:58 : Хеш-последовательность успешно получена
28.03.2021 13:57:58 : Дешифрование пароля успешно завершено
28.03.2021 13:57:58 : System.Exception: System.Exception: Npgsql.PostgresException (0x80004005): 42P01: отношение "public.PathArhive" не существует
   в Npgsql.NpgsqlConnector.<>c__DisplayClass160_0.<<DoReadMessage>g__ReadMessageLong|0>d.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlConnector.cs:строка 942
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   в Npgsql.NpgsqlConnector.<>c__DisplayClass160_0.<<DoReadMessage>g__ReadMessageLong|0>d.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlConnector.cs:строка 973
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   в System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   в System.Threading.Tasks.ValueTask`1.get_Result()
   в Npgsql.NpgsqlDataReader.<NextResult>d__44.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlDataReader.cs:строка 422
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   в System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   в Npgsql.NpgsqlDataReader.NextResult() в C:\projects\npgsql\src\Npgsql\NpgsqlDataReader.cs:строка 306
   в Npgsql.NpgsqlCommand.<ExecuteReaderAsync>d__102.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlCommand.cs:строка 1189
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   в System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   в Npgsql.NpgsqlCommand.ExecuteReader(CommandBehavior behavior) в C:\projects\npgsql\src\Npgsql\NpgsqlCommand.cs:строка 1059
   в Npgsql.NpgsqlCommand.ExecuteReader() в C:\projects\npgsql\src\Npgsql\NpgsqlCommand.cs:строка 1050
   в SZMK.Desktop.Services.Request.GetAllPathArhive() в C:\Users\thedr\Desktop\Works\SZMK\SZMK-Software-Package\SZMK.Desktop\Services\Request.cs:строка 2204
  Exception data:
    Severity: ОШИБКА
    SqlState: 42P01
    MessageText: отношение "public.PathArhive" не существует
    Position: 39
    File: d:\pginstaller_12.auto\postgres.windows-x64\src\backend\parser\parse_relation.c
    Line: 1173
    Routine: parserOpenTable
   в SZMK.Desktop.Services.Request.GetAllPathArhive() в C:\Users\thedr\Desktop\Works\SZMK\SZMK-Software-Package\SZMK.Desktop\Services\Request.cs:строка 2219
   в SZMK.Desktop.Services.Request.GetAllOrders() в C:\Users\thedr\Desktop\Works\SZMK\SZMK-Software-Package\SZMK.Desktop\Services\Request.cs:строка 1065 ---> System.Exception: System.Exception: Npgsql.PostgresException (0x80004005): 42P01: отношение "public.PathArhive" не существует
   в Npgsql.NpgsqlConnector.<>c__DisplayClass160_0.<<DoReadMessage>g__ReadMessageLong|0>d.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlConnector.cs:строка 942
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   в Npgsql.NpgsqlConnector.<>c__DisplayClass160_0.<<DoReadMessage>g__ReadMessageLong|0>d.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlConnector.cs:строка 973
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   в System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   в System.Threading.Tasks.ValueTask`1.get_Result()
   в Npgsql.NpgsqlDataReader.<NextResult>d__44.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlDataReader.cs:строка 422
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   в System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   в Npgsql.NpgsqlDataReader.NextResult() в C:\projects\npgsql\src\Npgsql\NpgsqlDataReader.cs:строка 306
   в Npgsql.NpgsqlCommand.<ExecuteReaderAsync>d__102.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlCommand.cs:строка 1189
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   в System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   в Npgsql.NpgsqlCommand.ExecuteReader(CommandBehavior behavior) в C:\projects\npgsql\src\Npgsql\NpgsqlCommand.cs:строка 1059
   в Npgsql.NpgsqlCommand.ExecuteReader() в C:\projects\npgsql\src\Npgsql\NpgsqlCommand.cs:строка 1050
   в SZMK.Desktop.Services.Request.GetAllPathArhive() в C:\Users\thedr\Desktop\Works\SZMK\SZMK-Software-Package\SZMK.Desktop\Services\Request.cs:строка 2204
  Exception data:
    Severity: ОШИБКА
    SqlState: 42P01
    MessageText: отношение "public.PathArhive" не существует
    Position: 39
    File: d:\pginstaller_12.auto\postgres.windows-x64\src\backend\parser\parse_relation.c
    Line: 1173
    Routine: parserOpenTable
   в SZMK.Desktop.Services.Request.GetAllPathArhive() в C:\Users\thedr\Desktop\Works\SZMK\SZMK-Software-Package\SZMK.Desktop\Services\Request.cs:строка 2219
   в SZMK.Desktop.Services.Request.GetAllOrders() в C:\Users\thedr\Desktop\Works\SZMK\SZMK-Software-Package\SZMK.Desktop\Services\Request.cs:строка 1065 ---> System.Exception: System.Exception: Npgsql.PostgresException (0x80004005): 42P01: отношение "public.PathArhive" не существует
   в Npgsql.NpgsqlConnector.<>c__DisplayClass160_0.<<DoReadMessage>g__ReadMessageLong|0>d.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlConnector.cs:строка 942
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   в Npgsql.NpgsqlConnector.<>c__DisplayClass160_0.<<DoReadMessage>g__ReadMessageLong|0>d.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlConnector.cs:строка 973
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   в System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   в System.Threading.Tasks.ValueTask`1.get_Result()
   в Npgsql.NpgsqlDataReader.<NextResult>d__44.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlDataReader.cs:строка 422
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   в System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   в Npgsql.NpgsqlDataReader.NextResult() в C:\projects\npgsql\src\Npgsql\NpgsqlDataReader.cs:строка 306
   в Npgsql.NpgsqlCommand.<ExecuteReaderAsync>d__102.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlCommand.cs:строка 1189
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   в System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   в Npgsql.NpgsqlCommand.ExecuteReader(CommandBehavior behavior) в C:\projects\npgsql\src\Npgsql\NpgsqlCommand.cs:строка 1059
   в Npgsql.NpgsqlCommand.ExecuteReader() в C:\projects\npgsql\src\Npgsql\NpgsqlCommand.cs:строка 1050
   в SZMK.Desktop.Services.Request.GetAllPathArhive() в C:\Users\thedr\Desktop\Works\SZMK\SZMK-Software-Package\SZMK.Desktop\Services\Request.cs:строка 2204
  Exception data:
    Severity: ОШИБКА
    SqlState: 42P01
    MessageText: отношение "public.PathArhive" не существует
    Position: 39
    File: d:\pginstaller_12.auto\postgres.windows-x64\src\backend\parser\parse_relation.c
    Line: 1173
    Routine: parserOpenTable
   в SZMK.Desktop.Services.Request.GetAllPathArhive() в C:\Users\thedr\Desktop\Works\SZMK\SZMK-Software-Package\SZMK.Desktop\Services\Request.cs:строка 2219
   в SZMK.Desktop.Services.Request.GetAllOrders() в C:\Users\thedr\Desktop\Works\SZMK\SZMK-Software-Package\SZMK.Desktop\Services\Request.cs:строка 1065
   в SZMK.Desktop.Services.Request.GetAllOrders() в C:\Users\thedr\Desktop\Works\SZMK\SZMK-Software-Package\SZMK.Desktop\Services\Request.cs:строка 1117
   в SZMK.Desktop.Services.RequestLinq.GetData() в C:\Users\thedr\Desktop\Works\SZMK\SZMK-Software-Package\SZMK.Desktop\Services\RequestLinq.cs:строка 307
   --- Конец трассировки внутреннего стека исключений ---
   в SZMK.Desktop.Services.RequestLinq.GetData() в C:\Users\thedr\Desktop\Works\SZMK\SZMK-Software-Package\SZMK.Desktop\Services\RequestLinq.cs:строка 313
   в SZMK.Desktop.Services.RequestLinq.GetOrdersUser(INotifyProcess notifier) в C:\Users\thedr\Desktop\Works\SZMK\SZMK-Software-Package\SZMK.Desktop\Services\RequestLinq.cs:строка 75
   --- Конец трассировки внутреннего стека исключений ---
   в SZMK.Desktop.Services.RequestLinq.GetOrdersUser(INotifyProcess notifier) в C:\Users\thedr\Desktop\Works\SZMK\SZMK-Software-Package\SZMK.Desktop\Services\RequestLinq.cs:строка 114
   в SZMK.Desktop.Views.KB.KB_F.GetOrders(Int32 ForViews, ForLongOperations_F Dialog) в C:\Users\thedr\Desktop\Works\SZMK\SZMK-Software-Package\SZMK.Desktop\Views\KB\KB_F.cs:строка 890
   в SZMK.Desktop.Views.KB.KB_F.RefreshOrder(Int32 ForViews, ForLongOperations_F Dialog) в C:\Users\thedr\Desktop\Works\SZMK\SZMK-Software-Package\SZMK.Desktop\Views\KB\KB_F.cs:строка 872
28.03.2021 14:04:05 : Дешифрование пароля успешно завершено
28.03.2021 14:04:11 : Хеш-последовательность успешно получена
28.03.2021 14:04:11 : Хеш-последовательность успешно получена
28.03.2021 14:04:11 : Дешифрование пароля успешно завершено
28.03.2021 14:04:11 : System.Exception: System.IndexOutOfRangeException: Column must be between 0 and 13
   в Npgsql.NpgsqlDataReader.CheckRowAndGetField(Int32 column) в C:\projects\npgsql\src\Npgsql\NpgsqlDataReader.cs:строка 1950
   в Npgsql.NpgsqlDataReader.IsDBNull(Int32 ordinal) в C:\projects\npgsql\src\Npgsql\NpgsqlDataReader.cs:строка 1533
   в SZMK.Desktop.Services.Request.GetAllOrders() в C:\Users\thedr\Desktop\Works\SZMK\SZMK-Software-Package\SZMK.Desktop\Services\Request.cs:строка 1101 ---> System.Exception: System.IndexOutOfRangeException: Column must be between 0 and 13
   в Npgsql.NpgsqlDataReader.CheckRowAndGetField(Int32 column) в C:\projects\npgsql\src\Npgsql\NpgsqlDataReader.cs:строка 1950
   в Npgsql.NpgsqlDataReader.IsDBNull(Int32 ordinal) в C:\projects\npgsql\src\Npgsql\NpgsqlDataReader.cs:строка 1533
   в SZMK.Desktop.Services.Request.GetAllOrders() в C:\Users\thedr\Desktop\Works\SZMK\SZMK-Software-Package\SZMK.Desktop\Services\Request.cs:строка 1101 ---> System.Exception: System.IndexOutOfRangeException: Column must be between 0 and 13
   в Npgsql.NpgsqlDataReader.CheckRowAndGetField(Int32 column) в C:\projects\npgsql\src\Npgsql\NpgsqlDataReader.cs:строка 1950
   в Npgsql.NpgsqlDataReader.IsDBNull(Int32 ordinal) в C:\projects\npgsql\src\Npgsql\NpgsqlDataReader.cs:строка 1533
   в SZMK.Desktop.Services.Request.GetAllOrders() в C:\Users\thedr\Desktop\Works\SZMK\SZMK-Software-Package\SZMK.Desktop\Services\Request.cs:строка 1101
   в SZMK.Desktop.Services.Request.GetAllOrders() в C:\Users\thedr\Desktop\Works\SZMK\SZMK-Software-Package\SZMK.Desktop\Services\Request.cs:строка 1117
   в SZMK.Desktop.Services.RequestLinq.GetData() в C:\Users\thedr\Desktop\Works\SZMK\SZMK-Software-Package\SZMK.Desktop\Services\RequestLinq.cs:строка 307
   --- Конец трассировки внутреннего стека исключений ---
   в SZMK.Desktop.Services.RequestLinq.GetData() в C:\Users\thedr\Desktop\Works\SZMK\SZMK-Software-Package\SZMK.Desktop\Services\RequestLinq.cs:строка 313
   в SZMK.Desktop.Services.RequestLinq.GetOrdersUser(INotifyProcess notifier) в C:\Users\thedr\Desktop\Works\SZMK\SZMK-Software-Package\SZMK.Desktop\Services\RequestLinq.cs:строка 75
   --- Конец трассировки внутреннего стека исключений ---
   в SZMK.Desktop.Services.RequestLinq.GetOrdersUser(INotifyProcess notifier) в C:\Users\thedr\Desktop\Works\SZMK\SZMK-Software-Package\SZMK.Desktop\Services\RequestLinq.cs:строка 114
   в SZMK.Desktop.Views.KB.KB_F.GetOrders(Int32 ForViews, ForLongOperations_F Dialog) в C:\Users\thedr\Desktop\Works\SZMK\SZMK-Software-Package\SZMK.Desktop\Views\KB\KB_F.cs:строка 890
   в SZMK.Desktop.Views.KB.KB_F.RefreshOrder(Int32 ForViews, ForLongOperations_F Dialog) в C:\Users\thedr\Desktop\Works\SZMK\SZMK-Software-Package\SZMK.Desktop\Views\KB\KB_F.cs:строка 872
28.03.2021 14:04:37 : Дешифрование пароля успешно завершено
28.03.2021 14:04:45 : Хеш-последовательность успешно получена
28.03.2021 14:04:45 : Хеш-последовательность успешно получена
28.03.2021 14:04:45 : Дешифрование пароля успешно завершено
28.03.2021 14:05:39 : Дешифрование пароля успешно завершено
28.03.2021 14:05:45 : Хеш-последовательность успешно получена
28.03.2021 14:05:45 : Хеш-последовательность успешно получена
28.03.2021 14:05:45 : Дешифрование пароля успешно завершено
28.03.2021 14:06:25 : Дешифрование пароля успешно завершено
28.03.2021 14:06:30 : Хеш-последовательность успешно получена
28.03.2021 14:06:30 : Хеш-последовательность успешно получена
28.03.2021 14:06:31 : Дешифрование пароля успешно завершено
28.03.2021 14:06:47 : System.Exception: Npgsql.PostgresException (0x80004005): 42703: столбец "PathDWG" не существует
   в Npgsql.NpgsqlConnector.<>c__DisplayClass160_0.<<DoReadMessage>g__ReadMessageLong|0>d.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlConnector.cs:строка 942
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   в Npgsql.NpgsqlConnector.<>c__DisplayClass160_0.<<DoReadMessage>g__ReadMessageLong|0>d.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlConnector.cs:строка 973
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   в System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   в System.Threading.Tasks.ValueTask`1.get_Result()
   в Npgsql.NpgsqlDataReader.<NextResult>d__44.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlDataReader.cs:строка 422
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   в System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   в Npgsql.NpgsqlDataReader.NextResult() в C:\projects\npgsql\src\Npgsql\NpgsqlDataReader.cs:строка 306
   в Npgsql.NpgsqlCommand.<ExecuteReaderAsync>d__102.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlCommand.cs:строка 1189
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   в System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   в Npgsql.NpgsqlCommand.ExecuteReader(CommandBehavior behavior) в C:\projects\npgsql\src\Npgsql\NpgsqlCommand.cs:строка 1059
   в Npgsql.NpgsqlCommand.ExecuteReader() в C:\projects\npgsql\src\Npgsql\NpgsqlCommand.cs:строка 1050
   в SZMK.Desktop.Services.Request.GetPathArhive(PathArhive PathArhive) в C:\Users\thedr\Desktop\Works\SZMK\SZMK-Software-Package\SZMK.Desktop\Services\Request.cs:строка 2286
  Exception data:
    Severity: ОШИБКА
    SqlState: 42703
    MessageText: столбец "PathDWG" не существует
    Hint: Возможно, предполагалась ссылка на столбец "PathArhive.Path".
    Position: 66
    File: d:\pginstaller_12.auto\postgres.windows-x64\src\backend\parser\parse_relation.c
    Line: 3359
    Routine: errorMissingColumn ---> System.Exception: Npgsql.PostgresException (0x80004005): 42703: столбец "PathDWG" не существует
   в Npgsql.NpgsqlConnector.<>c__DisplayClass160_0.<<DoReadMessage>g__ReadMessageLong|0>d.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlConnector.cs:строка 942
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   в Npgsql.NpgsqlConnector.<>c__DisplayClass160_0.<<DoReadMessage>g__ReadMessageLong|0>d.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlConnector.cs:строка 973
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   в System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   в System.Threading.Tasks.ValueTask`1.get_Result()
   в Npgsql.NpgsqlDataReader.<NextResult>d__44.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlDataReader.cs:строка 422
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   в System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   в Npgsql.NpgsqlDataReader.NextResult() в C:\projects\npgsql\src\Npgsql\NpgsqlDataReader.cs:строка 306
   в Npgsql.NpgsqlCommand.<ExecuteReaderAsync>d__102.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlCommand.cs:строка 1189
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   в System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   в Npgsql.NpgsqlCommand.ExecuteReader(CommandBehavior behavior) в C:\projects\npgsql\src\Npgsql\NpgsqlCommand.cs:строка 1059
   в Npgsql.NpgsqlCommand.ExecuteReader() в C:\projects\npgsql\src\Npgsql\NpgsqlCommand.cs:строка 1050
   в SZMK.Desktop.Services.Request.GetPathArhive(PathArhive PathArhive) в C:\Users\thedr\Desktop\Works\SZMK\SZMK-Software-Package\SZMK.Desktop\Services\Request.cs:строка 2286
  Exception data:
    Severity: ОШИБКА
    SqlState: 42703
    MessageText: столбец "PathDWG" не существует
    Hint: Возможно, предполагалась ссылка на столбец "PathArhive.Path".
    Position: 66
    File: d:\pginstaller_12.auto\postgres.windows-x64\src\backend\parser\parse_relation.c
    Line: 3359
    Routine: errorMissingColumn
   в SZMK.Desktop.Services.Request.GetPathArhive(PathArhive PathArhive) в C:\Users\thedr\Desktop\Works\SZMK\SZMK-Software-Package\SZMK.Desktop\Services\Request.cs:строка 2302
   в SZMK.Desktop.Views.KB.KB_DetailedInformationOrder_F.SetPathArhive(PathArhive oldpath) в C:\Users\thedr\Desktop\Works\SZMK\SZMK-Software-Package\SZMK.Desktop\Views\KB\KB_DetailedInformationOrder_F.cs:строка 617
   --- Конец трассировки внутреннего стека исключений ---
   в SZMK.Desktop.Views.KB.KB_DetailedInformationOrder_F.SetPathArhive(PathArhive oldpath) в C:\Users\thedr\Desktop\Works\SZMK\SZMK-Software-Package\SZMK.Desktop\Views\KB\KB_DetailedInformationOrder_F.cs:строка 635
   в SZMK.Desktop.Views.KB.KB_DetailedInformationOrder_F.ChangePathArhive_B_Click(Object sender, EventArgs e) в C:\Users\thedr\Desktop\Works\SZMK\SZMK-Software-Package\SZMK.Desktop\Views\KB\KB_DetailedInformationOrder_F.cs:строка 500
28.03.2021 14:07:25 : System.Exception: Не найдена папка
   в SZMK.Desktop.Views.KB.KB_DetailedInformationOrder_F.ChangePathArhive_B_Click(Object sender, EventArgs e) в C:\Users\thedr\Desktop\Works\SZMK\SZMK-Software-Package\SZMK.Desktop\Views\KB\KB_DetailedInformationOrder_F.cs:строка 511
28.03.2021 14:07:28 : System.Exception: Не найдена папка
   в SZMK.Desktop.Views.KB.KB_DetailedInformationOrder_F.ChangePathArhive_B_Click(Object sender, EventArgs e) в C:\Users\thedr\Desktop\Works\SZMK\SZMK-Software-Package\SZMK.Desktop\Views\KB\KB_DetailedInformationOrder_F.cs:строка 511
28.03.2021 14:07:36 : Дешифрование пароля успешно завершено
28.03.2021 14:07:41 : Хеш-последовательность успешно получена
28.03.2021 14:07:41 : Хеш-последовательность успешно получена
28.03.2021 14:07:41 : Дешифрование пароля успешно завершено
28.03.2021 14:07:51 : System.Management.ManagementException: Не найдено 
   в System.Management.ManagementException.ThrowWithExtendedInfo(ManagementStatus errorCode)
   в System.Management.ManagementObject.Initialize(Boolean getObject)
   в System.Management.ManagementBaseObject.get_Properties()
   в System.Management.ManagementBaseObject.GetPropertyValue(String propertyName)
   в System.Management.ManagementBaseObject.get_Item(String propertyName)
   в SZMK.Desktop.Views.KB.KB_DetailedInformationOrder_F.ChangePathArhive_B_Click(Object sender, EventArgs e) в C:\Users\thedr\Desktop\Works\SZMK\SZMK-Software-Package\SZMK.Desktop\Views\KB\KB_DetailedInformationOrder_F.cs:строка 523
28.03.2021 14:08:38 : Дешифрование пароля успешно завершено
28.03.2021 14:08:44 : Хеш-последовательность успешно получена
28.03.2021 14:08:44 : Хеш-последовательность успешно получена
28.03.2021 14:08:44 : Дешифрование пароля успешно завершено
28.03.2021 14:33:01 : Дешифрование пароля успешно завершено
28.03.2021 14:33:05 : Хеш-последовательность успешно получена
28.03.2021 14:33:05 : Хеш-последовательность успешно получена
28.03.2021 14:33:05 : Дешифрование пароля успешно завершено
28.03.2021 14:33:44 : Дешифрование пароля успешно завершено
28.03.2021 14:33:50 : Хеш-последовательность успешно получена
28.03.2021 14:33:50 : Хеш-последовательность успешно получена
28.03.2021 14:33:50 : Дешифрование пароля успешно завершено
28.03.2021 14:34:50 : Дешифрование пароля успешно завершено
28.03.2021 14:34:57 : Хеш-последовательность успешно получена
28.03.2021 14:34:57 : Хеш-последовательность успешно получена
28.03.2021 14:34:57 : Дешифрование пароля успешно завершено
28.03.2021 14:48:08 : Дешифрование пароля успешно завершено
28.03.2021 14:48:15 : Хеш-последовательность успешно получена
28.03.2021 14:48:16 : Хеш-последовательность успешно получена
28.03.2021 14:48:16 : Хеш-последовательность успешно получена
28.03.2021 15:01:06 : Дешифрование пароля успешно завершено
28.03.2021 15:01:11 : Хеш-последовательность успешно получена
28.03.2021 15:01:11 : Хеш-последовательность успешно получена
28.03.2021 15:01:11 : Дешифрование пароля успешно завершено
28.03.2021 17:57:32 : Дешифрование пароля успешно завершено
28.03.2021 17:57:38 : Хеш-последовательность успешно получена
28.03.2021 17:57:38 : Хеш-последовательность успешно получена
28.03.2021 17:57:38 : Дешифрование пароля успешно завершено
28.03.2021 17:58:03 : System.ArgumentOutOfRangeException: Индекс за пределами диапазона. Индекс должен быть положительным числом, а его размер не должен превышать размер коллекции.
Имя параметра: index
   в System.ThrowHelper.ThrowArgumentOutOfRangeException(ExceptionArgument argument, ExceptionResource resource)
   в System.Collections.Generic.List`1.get_Item(Int32 index)
   в System.Collections.ObjectModel.Collection`1.get_Item(Int32 index)
   в SZMK.Desktop.Views.KB.KB_ReportCheckDetail.Report_DGV_CellClick(Object sender, DataGridViewCellEventArgs e) в C:\Users\thedr\Desktop\Works\SZMK\SZMK-Software-Package\SZMK.Desktop\Views\KB\KB_ReportCheckDetail.cs:строка 69
   в System.Windows.Forms.DataGridView.OnCellClick(DataGridViewCellEventArgs e)
   в System.Windows.Forms.DataGridView.OnMouseClick(MouseEventArgs e)
   в System.Windows.Forms.Control.WmMouseUp(Message& m, MouseButtons button, Int32 clicks)
   в System.Windows.Forms.Control.WndProc(Message& m)
   в System.Windows.Forms.DataGridView.WndProc(Message& m)
   в System.Windows.Forms.Control.ControlNativeWindow.OnMessage(Message& m)
   в System.Windows.Forms.Control.ControlNativeWindow.WndProc(Message& m)
   в System.Windows.Forms.NativeWindow.DebuggableCallback(IntPtr hWnd, Int32 msg, IntPtr wparam, IntPtr lparam)
   в System.Windows.Forms.UnsafeNativeMethods.DispatchMessageW(MSG& msg)
   в System.Windows.Forms.Application.ComponentManager.System.Windows.Forms.UnsafeNativeMethods.IMsoComponentManager.FPushMessageLoop(IntPtr dwComponentID, Int32 reason, Int32 pvLoopData)
   в System.Windows.Forms.Application.ThreadContext.RunMessageLoopInner(Int32 reason, ApplicationContext context)
   в System.Windows.Forms.Application.ThreadContext.RunMessageLoop(Int32 reason, ApplicationContext context)
   в System.Windows.Forms.Application.RunDialog(Form form)
   в System.Windows.Forms.Form.ShowDialog(IWin32Window owner)
   в System.Windows.Forms.Form.ShowDialog()
   в SZMK.Desktop.Services.Scan.ParseXML.CheckDetails() в C:\Users\thedr\Desktop\Works\SZMK\SZMK-Software-Package\SZMK.Desktop\Services\Scan\ParseXML.cs:строка 375
   в SZMK.Desktop.Views.KB.KB_F.<StartingParseAsync>d__57.MoveNext() в C:\Users\thedr\Desktop\Works\SZMK\SZMK-Software-Package\SZMK.Desktop\Views\KB\KB_F.cs:строка 1410
28.03.2021 18:05:10 : Дешифрование пароля успешно завершено
28.03.2021 18:05:31 : Дешифрование пароля успешно завершено
28.03.2021 18:05:36 : Хеш-последовательность успешно получена
28.03.2021 18:05:36 : Хеш-последовательность успешно получена
28.03.2021 18:05:37 : Дешифрование пароля успешно завершено
28.03.2021 18:08:31 : Дешифрование пароля успешно завершено
28.03.2021 18:08:37 : Хеш-последовательность успешно получена
28.03.2021 18:08:37 : Хеш-последовательность успешно получена
28.03.2021 18:08:37 : Дешифрование пароля успешно завершено
28.03.2021 18:11:43 : Дешифрование пароля успешно завершено
28.03.2021 18:11:48 : Хеш-последовательность успешно получена
28.03.2021 18:11:48 : Хеш-последовательность успешно получена
28.03.2021 18:11:49 : Дешифрование пароля успешно завершено
28.03.2021 18:11:56 : Дешифрование пароля успешно завершено
28.03.2021 18:12:02 : Хеш-последовательность успешно получена
28.03.2021 18:12:02 : Хеш-последовательность успешно получена
28.03.2021 18:12:02 : Дешифрование пароля успешно завершено
28.03.2021 18:12:09 : System.Exception: Не найдена папка
   в SZMK.Desktop.Views.Shared.DetailedInformaionsOrder.PathArhive_TB_MouseDoubleClick(Object sender, MouseEventArgs e) в C:\Users\thedr\Desktop\Works\SZMK\SZMK-Software-Package\SZMK.Desktop\Views\Shared\DetailedInformaionsOrder.cs:строка 537
28.03.2021 18:12:23 : System.Exception: Не найдена папка
   в SZMK.Desktop.Views.Shared.DetailedInformaionsOrder.ChangePathDetailsDXF_B_Click(Object sender, EventArgs e) в C:\Users\thedr\Desktop\Works\SZMK\SZMK-Software-Package\SZMK.Desktop\Views\Shared\DetailedInformaionsOrder.cs:строка 254
28.03.2021 18:13:32 : Дешифрование пароля успешно завершено
28.03.2021 18:13:37 : Дешифрование пароля успешно завершено
28.03.2021 18:13:43 : Хеш-последовательность успешно получена
28.03.2021 18:13:43 : Хеш-последовательность успешно получена
28.03.2021 18:13:43 : Дешифрование пароля успешно завершено
28.03.2021 18:17:33 : Дешифрование пароля успешно завершено
28.03.2021 18:39:45 : Дешифрование пароля успешно завершено
28.03.2021 18:39:50 : Хеш-последовательность успешно получена
28.03.2021 18:39:50 : Хеш-последовательность успешно получена
28.03.2021 18:39:50 : Дешифрование пароля успешно завершено
28.03.2021 18:41:14 : Дешифрование пароля успешно завершено
28.03.2021 18:44:03 : Дешифрование пароля успешно завершено
28.03.2021 18:44:09 : Хеш-последовательность успешно получена
28.03.2021 18:44:09 : Хеш-последовательность успешно получена
28.03.2021 18:44:09 : Дешифрование пароля успешно завершено
28.03.2021 18:44:32 : System.Exception: Не найдена папка
   в SZMK.Desktop.Views.Shared.DetailedInformaionsOrder.PathDetailsDXF_TB_MouseDoubleClick(Object sender, MouseEventArgs e) в C:\Users\thedr\Desktop\Works\SZMK\SZMK-Software-Package\SZMK.Desktop\Views\Shared\DetailedInformaionsOrder.cs:строка 497
